smart_card_connector_app/src/reader-tracker.js

```diff
@@ -0,0 +1,212 @@
+/** @license
```

Comment:
I would suggest to move this JavaScript file under the third_party/pcsc-lite/naclport/server/src directory, because it provides a piece that is required for the PC/SC-Lite server to function correctly (because without this class, the messages sent from your new code would be left unhandled on the JS side).

And together with moving the file, add also something like "PcscLiteServer" into the provided symbols name, for better scoping.